blog.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="zxx">
+   <head>
+      <meta charset="utf-8" />
+      <title>ADOPTADOG</title>
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <link rel="stylesheet" type="text/css" href="styles/stylesheet.css" />
+      <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+      <link rel='stylesheet' href='https://fonts.googleapis.com/css?family=Roboto'>
+      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+      <link href='https://fonts.googleapis.com/css?family=Arvo' rel='stylesheet'>
+   </head>
+   <body>
+      <header>
+         <div class="header">
+            <div class="header-logo">
+               <img src="images/adoptadog-logo.svg" alt="Image of the Company Logo" />
+               <h2 class="slogan">Seattle's premiere dog adoption agency</h2>
+            </div>
+            <div class="shoppingcart">
+               <img src="images/cart-icon.svg" alt="Image of the Cart" />
+               <p>$1234.56</p>
+               <button class="checkout-button" onclick="document.location='checkout.html'">Checkout</button>
+            </div>
+         </div>
+         <nav>
+            <ul>
+               <li><a href="index.html">Home </a> </li>
+               <li><a href="dogs.html"> Dogs </a> </li>
+               <li><a href="blog.html"> Blog </a> </li>
+            </ul>
+         </nav>
+      </header>
+      <main>
+         <h3 class="blog-title">Adoptadog Blog</h3>
+         <div class="container-about">
+            <img src="images/blog-1.jpg" alt="Image of the blog-1" />
+            <div class="blog-post">
```

Comment:
You'll want to give your blog-post a percentage of left-margin to keep it spaced away from your paragraph text.